configure

```diff
@@ -288,6 +288,11 @@ parser.add_option('--enable-d8',
     dest='enable_d8',
     help=optparse.SUPPRESS_HELP)  # Unsupported, undocumented.
 
+parser.add_option('--enable-trace-maps',
+    action='store_true',
+    dest='trace_maps',
+    help='Enable the --trace-maps flag in v8')
```

Comment:
Consider undocumenting this or add a `(use at your own risk)` because it introduces an incompatibility between builds with and without the flag.

It adds a field to SharedFunctionInfo that ends up in the code cache so the result of `new vm.Script(source, { produceCachedData: true })` from a `-DTRACE_MAPS` build  won't be loadable in a normal build and vice versa.  I didn't test but I expect it segfaults.